test/functional/Java8andUp/src/org/openj9/test/java/security/DigestLoop.java

```diff
@@ -38,8 +39,9 @@ public final class DigestLoop extends Thread {
     static final int ITERATIONS = 1000000;
 
     static byte[] data;
-    static MessageDigest md;
     static Logger logger = Logger.getLogger(DigestLoop.class);
+    static final Object syncObj = new Object();
```

Comment:
I don't see the value of synchronization on `syncObj`. All it seems to do is ensure that each thread has started before the next thread is started. What would go wrong if `syncObj` were not added?

Reply:
That `join()` is only called after the loop that calls `start()` ensure that. I still don't see the need for `syncObj`.

Reply:
Just because start() returns doesn't mean the thread has started and is ready to be joined.

Reply:
If you really want me to remove it I will, because the chances of something going wrong for this test case are probably non-existent.

Reply:
> Just because start() returns doesn't mean the thread has started and is ready to be joined.

On the contrary, I think it does.

I don't think we should add `syncObj` unless there's a good reason.

Reply:
Updated.